templates/admin/repo/list.tmpl

```diff
@@ -7,7 +7,7 @@
 			</div>
 		</h4>
 		<div class="ui attached segment">
-			{{template "admin/repo/search" .}}
+			{{template "shared/repo_search" .}}
```

Comment:
Is it an over-abstraction?

Maybe admin page won't share the same filter as the end users.

Reply:
It was already like 99% identical code, now its just in one file
And for me its better this way, because I can change one file to have it changed everywhere and I also want to filter in my admin-view - maybe to see all mirrors or something like this

![image](https://github.com/go-gitea/gitea/assets/72873130/3097a7d1-50ae-4bf4-bf87-edad6e8069ab)

Reply:
But does the admin list support the newly added parameters? I didn't see related change.

Over-abstraction would cause many problems (eg: the one fixed by  #29430), because the "sort order" is shared by too many pages, it introduced undocumented parameter values later.

Reply:
Its working for me and it looks like the code was already ready to get this parameters
https://github.com/go-gitea/gitea/blob/bad4ad70181c747599e206c0e7a87b57c997385d/routers/web/admin/repos.go#L33C10-L33C26

Reply:
OK, then it could LGTM at the moment.